src/android/Notification.java

```diff
@@ -286,7 +286,9 @@ public synchronized void prompt(final String message, final String title, final
         Runnable runnable = new Runnable() {
             public void run() {
                 final EditText promptInput =  new EditText(cordova.getActivity());
-                promptInput.setHint(defaultText);
+                final int promptInputColor = 0xff444444;
```

Comment:
@matrosovN, i think you should avoid using 'magic' numbers. IMO this value should be read from resources, rather than hardcoded.